pkg/services/ngalert/notifier/channels/webex.go

```diff
@@ -0,0 +1,210 @@
+package channels
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/url"
+	"strings"
+
+	"github.com/prometheus/alertmanager/template"
+	"github.com/prometheus/alertmanager/types"
+
+	"github.com/grafana/grafana/pkg/infra/log"
+	"github.com/grafana/grafana/pkg/models"
+	ngmodels "github.com/grafana/grafana/pkg/services/ngalert/models"
+	"github.com/grafana/grafana/pkg/services/notifications"
+)
+
+const webexAPIURL = "https://webexapis.com/v1/messages"
+
+// WebexNotifier is responsible for sending alert notifications as webex messages.
+type WebexNotifier struct {
+	*Base
+	ns       notifications.WebhookSender
+	log      log.Logger
+	images   ImageStore
+	tmpl     *template.Template
+	orgID    int64
+	settings *webexSettings
+}
+
+// PLEASE do not touch these settings without taking a look at what we support as part of
+// https://github.com/prometheus/alertmanager/blob/main/notify/webex/webex.go
```

Comment:
Link doesn't work for me

Reply:
It's not there yet, we need to wait for https://github.com/prometheus/alertmanager/pull/3132 to be merged, which I'll merge on Monday once I get another quick review from alex.